src/main/java/sirius/db/jdbc/constraints/FieldOperator.java

```diff
@@ -9,32 +9,37 @@
 package sirius.db.jdbc.constraints;
 
 import sirius.db.jdbc.SmartQuery;
-import sirius.db.mixing.Mapping;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.commons.Tuple;
+
+import java.util.List;
 
 /**
  * Represents a simple field operator as constraint.
  */
 class FieldOperator extends SQLConstraint {
 
-    private Mapping field;
-    private Object value;
-    private String op;
+    private final RowValue lhs;
+    private final RowValue rhs;
```

Comment:
Optional: Können wir da evtl deskriptivere Feldnamen etc wählen. Also ich vermute mal das steht für left und right hand side oder? Aber so richtig sicher ist man sich da dann nach ner Weile evtl net mehr wenn man da in nem halben Jahr was dran basteln muss.